src/common/options/global.yaml.in

```diff
@@ -3622,6 +3622,9 @@ options:
   - memstore
   - kstore
   - seastore
+  - alienstore+bluestore
+  - alienstore+memstore
+  - alienstore+random
```

Comment:
what's the use case of "random"?

Reply:
The sole I can imagine is a teuthology testing.

Reply:
i'd use a `$` facet to do this.